composer.lock

```diff
@@ -544,6 +544,65 @@
             ],
             "time": "2021-12-08T12:19:24+00:00"
         },
+        {
+            "name": "phpstan/phpstan",
+            "version": "1.7.10",
+            "source": {
+                "type": "git",
+                "url": "https://github.com/phpstan/phpstan.git",
```

Comment:
Should composer.lock file be in the repository?

Reply:
Yes